src/Pelagos/Bundle/AppBundle/Controller/UI/PersonController.php

```diff
@@ -56,8 +58,13 @@ public function defaultAction($id = null)
                     $ui['PersonResearchGroupForms'][$personResearchGroup->getId()] = $formView;
                     $ui['PersonResearchGroupEditLabel'][$personResearchGroup->getId()]
                         = new EntityProperty($personResearchGroup, 'label');
+                    $listOfPersonsInResearchGroup = $personResearchGroup->getResearchGroup()->getPersonResearchGroups();
+                    foreach ($listOfPersonsInResearchGroup as $personResearch) {
+                        if ($personResearch->getRole()->getName() === ResearchGroupRoles::LEADERSHIP) {
+                            $personInLeadership = $personResearch->getPerson();
```

Comment:
since the ORM doesn't have any order specified in the PersonResearchGroup entity for PersonResearchGroups, there is no guarantee the first, last, or nth will will be selected. I know in almost all cases there is only one, but in the off case there isn't, it's just going to return any one (and may vary per fetch).

Reply:
Changed the logic to show multiple PI's if present